src/crypto/qat/QatDrv.cmake

```diff
@@ -0,0 +1,23 @@
+##
+# Make file for QAT linux driver project
+##
+
+set(qat_driver_dir ${CMAKE_SOURCE_DIR}/src/crypto/qat/qat-driver)
+set(qatdrv_include -I${Boost_INCLUDE_DIRS})
+
+include(ExternalProject)
+ExternalProject_Add(QatDrv
+  DEPENDS Boost
+  URL "${qat_driver_dir}/qat1.7.upstream.l.1.0.3-42.tar.gz"
+  URL_HASH SHA256=84d2c4a242987548135274da7c3def31461af6f1b4beb74f519a993e854abf5b
+  CONFIGURE_COMMAND CPPFLAGS=${qatdrv_include} CXXFLAGS=${qatdrv_include} ./configure --with-boost=${CMAKE_BINARY_DIR}/boost
```

Comment:
since QAT supports FreeBSD also. and the FreeBSD could use the port/package system for installing boost. see https://github.com/ceph/ceph/blob/master/install-deps.sh#L61 .

Reply:
thanks. Will move it to cmake/modules and make it find the path and use the sources as a fallback.